logtape/logger.ts

```diff
@@ -513,9 +513,11 @@ export class LoggerImpl implements Logger {
   logLazily(
     level: LogLevel,
     callback: LogCallback,
+    properties: Record<string, unknown> | (() => Record<string, unknown>) = {},
+    bypassSinks?: Set<Sink>,
```

Comment:
I think `bypassSinks` is kind of an internal parameter, so it's probably okay not to expose it.

Reply:
Gotcha